static/app/components/events/traceEventDataSection.tsx

```diff
@@ -287,7 +289,8 @@ const StyledIconAnchor = styled(IconAnchor)`
 const Permalink = styled('a')`
   display: inline-flex;
   justify-content: flex-start;
-  :hover ${StyledIconAnchor} {
+
+  &:hover ${StyledIconAnchor} {
```

Comment:
please, what is the difference between `&:hover ` and ` :hover`?

Reply:
@k-fish I actually replaced it thinking that it is broken and that it probably doesn't compile correctly 😅

Reply:
oh, I see... can we enforce this through eslint, so we can keep the consistency? and thanks for the explanation @k-fish 🙏